package.json

```diff
@@ -13,5 +13,6 @@
     "install:tests": "cd ./workspaces/tests && pnpm install",
     "install:calendar-widgets": "cd ./workspaces/calendar-widgets && pnpm install",
     "new": "pnpm run install:tests && pnpm run install:calendar-widgets && pnpm run install:docs"
-  }
+  },
+  "license": "MIT"
```

Comment:
This should be relocated to the `package.json` file within the `workspaces/calendar-widgets/` directory. That way, when we publish the package to NPM, the license will be found in the distributable code as well 🙂

Reply:
Thanks for the guidance. As suggested, I've relocated the license information to workspaces/calendar-widgets/package.json.